doc/FAQ.org

```diff
@@ -282,6 +305,28 @@ To restore the sentence delimiter to two spaces, add the following code to the
 (setq sentence-end-double-space t)
 #+end_src
 
+** Change the initial mode of the =*scratch*= buffer?
+To change the initial mode of the =*scratch*= buffer, add the following code to
+the =dotspacemacs/user-config= function of your =.spacemacs=:
+
+#+begin_src emacs-lisp
+(setq initial-major-mode 'ruby-mode)
+#+end_src
+
+Change =ruby-mode= accordingly to your needs.
+
+** Prevent the visual selection to override my system clipboard?
+On some operating systems, there is only one clipboard for both *copied* and
+*selected* texts. This has for consequence that visual *selection* – which
+should normally be saved to the ~PRIMARY~ clipboard – overrides the ~SYSTEM~
```

Comment:
primary and system aren't keys. Just emphasise the normal way: `/primary/` or `/system/`.